modules/ROOT/pages/configure-client-management-openid-task.adoc

```diff
@@ -47,7 +47,39 @@ The *External Identity* page appears.
  In *Advanced Settings* you can also select:
   *** *Disable server certificate validation*: Disables server certificate validation if your OpenID client management instance presents a self-signed certificate, or one signed by an internal certificate authority.
   *** *Enable client deletion in Anypoint Platform*: Enables deletion of clients created with this integration.
-  *** *Enable client deletion and updates in IdP*: To use this option, you must also select the *Enable client deletion in Anypoint Platform* option. This option enables you to update and delete external clients in the configured IdP through an outbound call made by Anypoint Platform. 
+  *** *Enable client deletion and updates in IdP*: To use this option, you must also select the *Enable client deletion in Anypoint Platform* option. This option enables you to update and delete external clients in the configured IdP through an outbound call made by Anypoint Platform to `{clientRegistrationUrl}/{clientID}`, for example: `+https://example.salesforce.com/services/oauth2/6779ef20e75817b79602+`. 
```

Comment:
I will also be good to let them know that `https://example.salesforce.com/services/oauth2/` is the client registration URL they configured and `6779ef20e75817b79602` is the clientId

Reply:
I think since it's an example of {clientRegistrationUrl}/{clientID}, it's clear that 6779ef20e75817b79602 is the client ID.